bridges/SIMARBridge.php

```diff
@@ -0,0 +1,63 @@
+<?php
+class SIMARBridge extends BridgeAbstract {
+	const NAME = 'SIMAR';
+	const URI = 'http://www.simar-louresodivelas.pt/';
+	const DESCRIPTION = 'Verificar estado da rede SIMAR';
+	const MAINTAINER = 'somini';
+	const PARAMETERS = array(
+		'Público' => array(
+			'interventions' => array(
+				'type' => 'checkbox',
+				'name' => 'Incluir Intervenções?',
+				'defaultValue' => 'checked',
+			)
+		)
+	);
+
+	public function collectData() {
+		$html = getSimpleHTMLDOM(self::getURI())
+			or returnServerError('Could not load content');
+		$e_home = $html->find('#home', 0)
+			or returnServerError('Invalid site structure');
+
+		foreach($e_home->find('span') as $element) {
+			$item = array();
+
+			$item['title'] = 'Rotura: ' . $element->plaintext;
+			$item['content'] = $element->innertext;
+			$item['uri'] = 'urn:sha1:' . hash('sha1', $item['content']);
+
+			$this->items[] = $item;
+		}
+
+		if ($this->getInput('interventions')) {
+			$e_main1 = $html->find('#menu1', 0)
+				or returnServerError('Invalid site structure');
+
+			foreach ($e_main1->find('a') as $element) {
+				$item = array();
+
+				$item['title'] = 'Intervenção: ' . $element->plaintext;
+				$item['uri'] = self::getURI() . $element->href;
+				$item['content'] = $element->innertext;
+
+				/* Try to get the actual contents for this kind of item */
+				$item_html = getSimpleHTMLDOM($item['uri']);
```

Comment:
Do you think it make sense to cache the contents, or do they change on every request?

```suggestion
				$item_html = getSimpleHTMLDOMCached($item['uri']);
```

Reply:
They don't change, otherwise they will get a different hash. Thanks for the heads-up.

Reply:
Oops, this is the interventions. But the contents shouldn't change.